content/training/RStudio_overview.Rmd

```diff
@@ -0,0 +1,30 @@
+---
+date: "2019-07-30T13:17:51+02:00"
+title: Orientation to RStudio
+weight: 5
+---
+
+Open RStudio. For RStudio to function you must also have R installed on the computer (see the X page for installation instructions). RStudio is an interface (GUI) for easier use of R. You can think of R as being the engine of a vehicle, doing the crucial work, and RStudio as the body of the vehicle (with seats, accessories, etc.) that helps you actually use the engine to move forward!
+
+By default RStudio displays four rectangle panes. 
+
+![](/images/RStudio_overview.png)
+*If your RStudio displays only one left pane it is because you have no scripts open yet. See below how to open or start a new script.*
+
+
+
+##The R Console Pane
```

Comment:
Please give the markdown some room to breathe. hashes and text should always be separated by a space  and there should be a newline after section headers.

Reply:
I added the spaces and newlines - please let me know if there places you were referring to that I didn't address

Reply:
That wasn't *exactly* what I was addressing. When I said to add newlines, I meant actual newlines:

so go from this

```
##Section header
text text text

##Section two
text text text
```

to this


```
## Section header

text text text

## Section two

text text text
```

Reply:
The slashes you inserted will not do anything. You might be thinking of LaTeX.